stand/efi/libefi32/Makefile

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
extra blank line

Reply:
Isn't a blank line left at the top of the other Makefiles? (At least, the ones that don't start with comments)
If that's not an enforced style, then yeah I'll remove it.

Reply:
Ah, just saw e9ac41698b2f322d55ccf9da50a3596edb2c1800.
Guess I confused that for a style.